### DISCUSSION TITLE
Anyone know if the Exlene GC controllers have the same multi input lag bug as other switch controllers?

### DISCUSSION THREAD

**Discussion by icd2k3** [darthcloud/BlueRetro#695] [Q&A] [answered]
I was looking at getting 4 PowerA GC controllers, but found out there is a bug with multiple switch controllers connecting to 1 BlueRetro board https://github.com/darthcloud/BlueRetro/issues/146

Since the Exlene [connects as xinput](https://github.com/darthcloud/BlueRetro/wiki/Controller-pairing-guide#12---exlene-gamecube-paring-guide) is it fine to connect multiple without lag?

**Comment by darthcloud** [comment, marked as answer]
not tested i only got one, but i would expect xinput to be fine.

It a switch mode issue only

**Reply by icd2k3** [reply]
Happy to report that 4 Exlene GC controllers work 100% fine - just select the preset for each output and you're good to go! No for any controller.

Really cool to be able to play GC wirelessly with 4 controllers that feel close to the original

**Comment by icd2k3** [comment]
Thanks @darthcloud - good enough for me! I'll take the risk and order a few of them and report back once I have everything installed and tested
